<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Analyzer</class>
 <widget class="QMainWindow" name="Analyzer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Analyzer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Open</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Analyze</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string> Analyzer</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QTextEdit" name="textEdit">
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>10</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableView" name="tableView">
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="gridStyle">
               <enum>Qt::SolidLine</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Pseudocode</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QTableView" name="tableView_2">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="gridStyle">
             <enum>Qt::SolidLine</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>Control Flow Graph</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Basic Blocks</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableView" name="tableView_4">
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="gridStyle">
               <enum>Qt::SolidLine</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Edges</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableView" name="tableView_3">
              <property name="font">
               <font>
                <family>Consolas</family>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="gridStyle">
               <enum>Qt::SolidLine</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>User manual</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>About program</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
